srcpkgs/ncdc/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ncdc'
+pkgname=ncdc
+version=1.20
+revision=1
+build_style=configure
```

Comment:
why not use `gnu-configure`? The package seems to use GNU autotools.

Reply:
it's works just as configure, so idk. I set gnu-configure now